src/PositionManager.sol

```diff
@@ -277,4 +320,10 @@ contract PositionManager is
             permit2.transferFrom(payer, address(poolManager), uint160(amount), Currency.unwrap(currency));
         }
     }
+
+    /// @dev overrides solmate transferFrom in case a notification to subscribers is needed
+    function transferFrom(address from, address to, uint256 id) public override {
```

Comment:
is it right that ERC721 doesnt have a `transfer` function?! Just `transferFrom`?? How am I just learning this

Reply:
yeah